### PR TITLE
initial commit to add pyntacle

### PR DESCRIPTION
**Pull request by AdMax18**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by AdMax18** [comment]
@mxr-conda please comment to be maintainer

File: recipes/pyntacle/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    - pip check
+    - pyntacle --help
+  requires:
+    - pip
+
+about:
+  home: http://pyntacle.css-mendel.it
+  summary: A Python package for network analysis based on non-canonical metrics and HPC-Computing
+  dev_url: https://github.com/mazzalab/pyntacle
+  license: GPL-3.0-only
```
Comment (dopplershift) [inline recipes/pyntacle/meta.yaml:51]: Looks like it [should be](https://github.com/mazzalab/pyntacle/blob/7d1d89c24262dcfa6cb2e4a68112a461fd020dc5/algorithms/global_topology.py#L16):
```suggestion
  license: GPL-3.0-or-later
```

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just a tweak to the license metadata and this is good to go.

**Review by dopplershift — APPROVED** [review]
Pending CI.

**Pull request merged**